Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -477,6 +477,7 @@
 		59D1E6581D1866F800896A4C /* DashboardVideoCellViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 59D1E6571D1866F800896A4C /* DashboardVideoCellViewModel.swift */; };
 		59E877381DC9419700BCD1F7 /* Newsletter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 59E877371DC9419700BCD1F7 /* Newsletter.swift */; };
 		6008633829BA51CD00B87B39 /* MockAppTrackingTrasparency.swift in Sources */ = {isa = PBXBuildFile; fileRef = 609C7DF029B67DC4008D66F2 /* MockAppTrackingTrasparency.swift */; };
+		6008633929BA61CE00B87B39 /* MockAppTrackingTrasparency.swift in Sources */ = {isa = PBXBuildFile; fileRef = 609C7DF029B67DC4008D66F2 /* MockAppTrackingTrasparency.swift */; };
```

Comment:
Please remove this reference, I think its' the old file misspelled fileXthat accidently got included.